client/src/pages/private/ParticipantTable.js

```diff
@@ -432,7 +434,7 @@ const ParticipantTable = () => {
           'withdrawn',
           'archived',
           'reject_ack',
-        ].find((status) => row.status.includes(status));
+        ].find((s) => row.status.includes(s));
```

Comment:
why this change? I think status is a more readable variable naming.

Reply:
Me too. This was because of a code smell detected by SonarCloud- the variable _status_ had already been declared in the upper scope. Do you think engagedStatus is accurate?